lib/data/file_manager/file_manager.dart

```diff
@@ -372,7 +372,10 @@ class FileManager {
   }
 
   static Future<DirectoryChildren?> getChildrenOfDirectory(
-      String directory) async {
+      String directory,
+      [bool removeExtension = true]) async {
```

Comment:
```suggestion
      {required bool removeExtension}) async {
```
I'd prefer a named parameter in this case as otherwise it's hard to read (will need some refactoring of course)